entity-management-tests/src/integration-test/resources/solr-docker/conf/schema.xml

```diff
@@ -264,6 +264,10 @@
 			stored="true" multiValued="true" />
 		<field name="skos_inScheme" type="string" indexed="true"
 			stored="true" multiValued="true" />
+		<field name="as_items" type="string" indexed="true"
```

Comment:
probably not needed in solr

Reply:
but other fields like subject and deifnition might need to be added, need to be properly defined in the Entity Collection Fields spreadsheet